qemu/tests/cfg/virtio_fs_supp_group_transfer.cfg

```diff
@@ -0,0 +1,50 @@
+- virtio_fs_supp_group_transfer:
+    type = virtio_fs_supp_group_transfer
+    only Linux
+    no Host_RHEL.m8, Host_RHEL.m7, Host_RHEL.m6
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    s390, s390x:
+        required_qemu = [5.2.0,)
+        vm_mem_share = yes
+        pre_command_noncritical = yes
+        pre_command = "echo 3 > /proc/sys/vm/drop_caches"
+        setup_hugepages = yes
+        kvm_module_parameters = 'hpage=1'
+        expected_hugepage_size = 1024
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    kill_vm = yes
+    start_vm = yes
+    filesystems = fs
+    fs_driver = virtio-fs
+    force_create_fs_source = yes
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    fs_driver_props = {"queue-size": 1024}
+    mem_devs = mem1
+    backend_mem_mem1 = memory-backend-file
+    mem-path_mem1 = /dev/shm
+    size_mem1 = ${mem}M
+    use_mem_mem1 = no
+    share_mem = yes
+    !s390, s390x:
+        guest_numa_nodes = shm0
+        numa_memdev_shm0 = mem-mem1
+        numa_nodeid_shm0 = 0
+    io_timeout = 600
+    fs_dest = '/mnt/${fs_target}'
+    new_guest_user = "user00001"
+    add_user_cmd = "useradd %s"
+    del_user_cmd = "userdel -r -f %s"
+    fs_enable_debug_mode = yes
+    fs_source_dir = virtio_fs_test/
+    variants:
+        - default:
+        - with_nfs_source:
+            setup_local_nfs = yes
```

Comment:
need to set start_vm = no here.
Setup nfs first and then start virtiofsd and vm.